dstep/main.d

```diff
@@ -63,7 +63,9 @@ auto parseCLI (string[] args)
         "no-comments", "Disable translation of comments.", &config.noComments,
         "public-submodules", "Use public imports for submodules.", &config.publicSubmodules,
         "package", "Specify package name.", &config.packageName,
-        "dont-reduce-aliases", "Disable reduction of primitive type aliases.", &config.dontReduceAliases);
+        "dont-reduce-aliases", "Disable reduction of primitive type aliases.", &config.dontReduceAliases,
+        "single-line-function-headers", "Do not break function headers to multiple lines.", &config.singleLineFunctionHeaders,
+        "no-space-after-function-name", "Do not put a space after a function name.", &config.noSpaceAfterFunctionName);
```

Comment:
It cannot be done automatically with getopt. The only way I see is to add a negation of given option to `Configuration` and check the constraints in code. E.g.

```
 "no-space-after-function-name", "Do not put a space after a function name.", &config.noSpaceAfterFunctionName,
 "space-after-function-name", "Put a space after a function name.", &config.spaceAfterFunctionName);

/+ ... +/

if (config.noSpaceAfterFunctionName && config.spaceAfterFunctionName) {
    // very bad
}
```